episodecard.js

```diff
@@ -0,0 +1,44 @@
+//function that would display episodes on page
+export default function createEpisodeCard(allEpisodes) {
+  const episodeList = document.createElement("ul"); // Episode list
+  episodeList.classList.add("main"); //main class attached to list
+  allEpisodes.forEach((episode) => {
+    //for each episode a li card is created
+    const lI = document.createElement("li");
+    lI.className = "episode-card";
+    const name = episode.name,
+      //individual episode data
+      season = episode.season,
```

Comment:
Great work Anza I like how you separate this entire function into different file. Very organised and your code are clean and easy to read.